package io.github.axion014.forgelang.analyze.exception;

public class CompilerInternalException extends RuntimeException {
	public CompilerInternalException(String message) {
		super(message);
	}
}
